### PR TITLE
Improve set/get forwarding pipeline config errors

### PR DESCRIPTION
**Pull request by daniele-moro**
- Don't allow the `setForwardingPipelineConfig`, generating `UNIMPLEMENTED` error as mentioned in the specs
- Return `FAILED_PRECONDITION` error if attempting write/read before we have a forwarding pipeline config (i.e., before UP4 netcfg is pushed)
- Return `FAILED_PRECONDITION` error when trying a `getForwadingPipelineConfig` before forwarding pipeline is ready. This is not fully p4rt-compliant (we should return empty config), but it makes it easier to manage this case in pfcp-agent

Required for: https://github.com/omec-project/upf/pull/547

### CODE REVIEW

File: .env.stable
Comment (daniele-moro) [inline .env.stable:5]: To be changed, added to test this change with https://github.com/omec-project/upf/pull/547

**Review by pierventre — APPROVED** [review]

**Comment by daniele-moro** [comment]
Wait for the release before merging.

**Pull request merged**
